content/learning-paths/servers-and-cloud-computing/new-java/2-deploy-java.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Install the JDK and build an application
+weight: 4
+
+### FIXED, DO NOT MODIFY
+layout: learningpathall
+---
+
+
+### You have two deployment options for Java on Azure Linux:
+
+ 1. Use the [Azure Linux 3.0 Docker container](https://learn.microsoft.com/en-us/azure/azure-linux/intro-azure-linux) on any supported platform.
+ 2. Use a custom Azure Linux ARM64 VM, as shown below.
+
+{{% notice Note %}}
+You must create an Azure Linux ARM VM using a custom VHD, since Azure Linux 3.0 ARM64 images are not directly available in the marketplace. Follow [this guide to convert the ISO to a VHD](https://github.com/ArmDeveloperEcosystem/arm-learning-paths/pull/2103) and deploy it as a VM.
+{{% /notice %}}
+
+
+### Working inside Azure Linux 3.0 Docker container
```

Comment:
This guide does not focus on the Docker container. It gives option to the user between docker container and custom VM. Edit the sentences to make the guide generic, not docker focused. Also, mention that the installation and benchmarking commands gonna be same on both platforms.